app/views/media_objects/_lti_url.html.erb

```diff
@@ -1,14 +1,14 @@
 <% if section=='tab-content' %>
 
     <div class="tab-pane" id="lti-tab">
-      <p class="muted">Copy the link below to share this resource via LTI</p>
+      <p class="muted"><%= t('share.lti.copy_link') %></p>
       <form role="form">
         <div class="form-group">
-          <label for="ltilink-object">Item</label>
+          <label for="ltilink-object"><%= t('share.item') %></label>
```

Comment:
The 'Item' and 'Section' labels also appear in the 'Share this resource' tab and aren't specific to LTI/eClass. The other labels have text that is LTI specific.